Assets/Scripts/ShopButtonController.cs

```diff
@@ -0,0 +1,174 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class ShopButtonController : MonoBehaviour
+{
+    float selection;
+    [Space(5)]
+    [Header("Pistol")]
+    public GameObject pistolSprite; //declaring pistol button sprite (default)
```

Comment:
as we discussed this might have been better as an array so you wouldn't have so much duplicate code below.